modules/analysis/rosbag-scene-detection/infrastructure/ecs_stack.py

```diff
@@ -119,29 +118,6 @@ def __init__(
             throughput_mode=efs.ThroughputMode.BURSTING,
             performance_mode=efs.PerformanceMode.MAX_IO,
         )
-        cfn_fs = cast(efs.CfnFileSystem, fs.node.default_child)
-        cfn_fs.file_system_policy = aws_iam.PolicyDocument(
-            statements=[
-                aws_iam.PolicyStatement(
-                    effect=aws_iam.Effect.ALLOW,
-                    principals=[aws_iam.AnyPrincipal()],
-                    actions=[
-                        "elasticfilesystem:ClientMount",
-                        "elasticfilesystem:ClientWrite",
-                        "elasticfilesystem:ClientRootAccess",
-                    ],
-                    resources=["*"],
-                    conditions={"Bool": {"elasticfilesystem:AccessedViaMountTarget": "true"}},
-                ),
-                aws_iam.PolicyStatement(
-                    effect=aws_iam.Effect.DENY,
-                    principals=[aws_iam.AnyPrincipal()],
-                    actions=["*"],
-                    resources=["*"],
-                    conditions={"Bool": {"aws:SecureTransport": "false"}},
-                ),
-            ]
-        )
```

Comment:
this file system policy was added to specifically address the Shepherd security findings ticket previously raised. Removing these will result in Shepherd raising security tickets in Conduit managed accounts.

Reply:
putting this back as it is 👍